internal/archive/archive.go

```diff
@@ -79,6 +87,23 @@ func (a *ubuntuArchive) Exists(pkg string) bool {
 	return err == nil
 }
 
+type pkgInfo struct{ control.Section }
+
+var _ PackageInfo = pkgInfo{}
+
+func (info pkgInfo) Name() string    { return info.Get("Package") }
+func (info pkgInfo) Version() string { return info.Get("Version") }
+func (info pkgInfo) Arch() string    { return info.Get("Architecture") }
+func (info pkgInfo) SHA256() string  { return info.Get("SHA256") }
+
+func (a *ubuntuArchive) Info(pkg string) (info PackageInfo) {
+	section, _, _ := a.selectPackage(pkg)
+	if section != nil {
+		info = &pkgInfo{section}
+	}
+	return
+}
```

Comment:
Naming the return value here does not yield much valuable information, in my opinion, and I'd rather avoid empty `return` statements. Also, since `section` is only used within the `if` body, it can be simplified to:

```suggestion
func (a *ubuntuArchive) Info(pkg string) PackageInfo {
	if section, _, _ := a.selectPackage(pkg); section != nil {
		return &pkgInfo{section}
	}
	return nil
}
```